docs/systemd/polylith-example@.service

```
@@ -0,0 +1,14 @@
+[Unit]
+	Description= Dendrite Polylith Multi - %I
+	PartOf=polyDendrite.service
+	After=network.target
+
+	[Service]
+	User=dendrite
+	WorkingDirectory=/opt/dendrite
+	Type=forking
+	ExecStart=/opt/dendrite/bin/dendrite-polylith-multi --config=dendrite.yaml %i
+	Restart=on-failure
```

```suggestion
	Restart=on-failure
	RestartSec=5s
```

Simply to let Kafka, Postgres or other components recover and not "DOS" them back to a failure state. :P